src/imars3d/ui/stages/dataloading.py

```diff
@@ -1,129 +1,114 @@
 #!/usr/bin/env python3
-
-import os
-import numpy as np
-import tifffile
+"""
+Data loading stage for iMars3D
+"""
 from pathlib import Path
 import param
 import panel as pn
-from panel.widgets import Tqdm
+from imars3d.backend.io.config import save_config
 
 
 class DataLoader(param.Parameterized):
-    # input from previous step
-    data_root = param.Path(
-        default=Path.home(),
-        doc="ct, ob, and df root directory, default should be proj_root/raw",
-    )
-    proj_root = param.String(doc="map data_root to string to avoid a FileSelector error")
-    recn_root = param.Path(
-        default=Path.home(),
-        doc="reconstruction results root, default should be proj_root/processed_data",
-    )
-    temp_root = param.Path(
-        default=Path.home() / Path("tmp"),
-        doc="intermedia results save location",
-    )
-    recn_name = param.String(
-        default="myrecon",
-        doc="reconstruction results folder name",
-    )
-    # data container
-    ct = param.Array(
-        doc="radiograph stack as numpy array",
-        precedence=-1,  # hide
+    # take over config from previous step
+    config_dict = param.Dict(
+        default={
+            "facility": "TBD",
+            "instrument": "TBD",
+            "ipts": 0,
+            "projectdir": "TBD",
+            "name": "TBD",
+            "workingdir": "TBD",
+            "outputdir": "TBD",
+            "steps": [],
```

Comment:
can you replace `"steps"` with `"tasks"` so that we use jargon from the Workflows world?

Reply:
I will update the name in the next commit